dev/PushNotifications/PushNotificationsLongRunningTask/NotificationListenerManager.cpp

```diff
@@ -15,30 +18,31 @@ void NotificationListenerManager::Initialize(std::shared_ptr<ForegroundSinkManag
 
 void NotificationListenerManager::AddListener(std::wstring appId, std::wstring processName)
```

Comment:
is appId case-sensitive?

Reply:
AppId here is just guid as a string so it shouldn't have to worry about being case-sensitive.